cupy/core/core.pyx

```diff
@@ -145,7 +145,7 @@ cdef class ndarray:
             'version': 0,
         }
         if not self._c_contiguous:
-            desc['strides'] = self._strides
+            desc['strides'] = tuple(self._strides)
```

Comment:
Maybe you should just use `self.strides`? Look above, they are just using `self.shape`. I know, your proposed fix is nanoseconds faster, but...

Reply:
@dalcinl good point. I didn't even think about speed. I was just thinking "I want a tuple, give me a tuple..." 😆